modules/Stories/StoryCard.tsx

```diff
@@ -76,7 +76,7 @@ const StoryCard: FunctionComponent<Props> = ({ story }) => {
                         <div className="mt-6 flex items-center">
                             <div className="flex space-x-1 text-sm text-gray-500">
                                 <time dateTime="2020-03-16">
```

Comment:
Why is the date hardcoded here? 😄  I'd say that if `publicationDate` is undefined then it doesn't make sense to render this element at all. And if it is defined, then probably render the correct date there.